tasks/shared_library_posting_to_queue/shared_lib_posting_to_status_queue.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+import json
+import boto3
+from typing import Dict,Any, Optional
+import time
+
+class RequestMethod(Enum):
+    POST = 'post'
+    PUT = 'put'
+
+class ORCA_STATUS(Enum):
+    """
+    Defines the status of ORCA DB?
+    """
+    PENDING = 1
+    STAGED = 2
+    SUCCESS = 3
+    FAILED = 4
+
+def post_status_for_job_to_queue(job_id: str, granule_id: str, status_id: ORCA_STATUS,
```

Comment:
@AndrewDorn need help from you on providing this info for all the 3 functions.

Reply:
@bhazuka, maybe we hold off on that until we revamp this code? A LOT of stuff is going to change.

Reply:
At a minimum, I would add a description of what the function is supposed to do and stub out the Args area. It would be ok not to put in argument definitions if we know that is volatile now. But the overall description should be fairly stable.